etc/spark-jobserver.conf

```diff
@@ -1,18 +1,18 @@
-spark {
-  master = "local[4]"
-
-  job-number-cpus = 4
-
-  jobserver {
-    port = 8090
-    jar-store-rootdir = /opt/spark-jobserver/jars
+###################
+# spark-jobserver #
+###################
 
-    jobdao = spark.jobserver.io.JobFileDAO
+spark.jobserver {
+  port = 8090
+  jar-store-rootdir = /opt/spark-jobserver/jars
+  filedao.rootdir = /opt/spark-jobserver/filedao/data
+}
 
-    filedao {
-      rootdir = /opt/spark-jobserver/filedao/data
-    }
-  }
+#########
+# spark #
+#########
 
+spark {
   home = "/opt/spark"
+  master = "local[4]"
```

Comment:
Addressed in https://github.com/azavea/docker-spark-jobserver/commit/7b59a3cc2c5f32cdf69fe8556dcad1463cd26577.